src/test/FuzzerImpl.cpp

```diff
@@ -810,6 +810,46 @@ generator_t::operator()(stellar::PublicKey& t) const
         t.ed25519(), static_cast<uint8_t>(stellar::rand_uniform<int>(
                          0, NUMBER_OF_ACCOUNT_IDS_TO_GENERATE - 1)));
 }
+
+static int RECURSION_COUNT = 0;
+static const int RECURSION_LIMIT = 50;
+
+template <>
+void
+generator_t::operator()<stellar::SCVal>(stellar::SCVal& val) const
+{
+    if (++RECURSION_COUNT > RECURSION_LIMIT)
+    {
+        stellar::SCVal v;
+        val = v;
+        return;
+    }
+    const auto& vals = stellar::SCVal::_xdr_case_values();
+    stellar::SCValType v;
+
+    uint32_t n;
```

Comment:
Why is `n` not initialized here?

Reply:
It gets set on the next line (I copied this from [xdrpp](https://github.com/xdrpp/xdrpp/blob/8756126cc734c11b75e05f174ca35a192860d0a6/xdrpp/autocheck.h#L118)).

Reply:
That's not really obvious - adding `=0` or `{}` wouldn't really hurt (also I'm pretty sure there is a warning for this).

Reply:
Updated